service/src/main/java/uk/gov/hmcts/reform/fpl/enums/cfv/DocumentType.java

```diff
@@ -208,8 +217,6 @@ public enum DocumentType {
     @Getter
     private Function<ConfidentialLevel, String> baseFieldNameResolver;
     @Getter
-    private boolean hiddenFromUpload;
```

Comment:
think the change looks ok to me. what was this flag used for before? just wondering why we're removing it now?

Reply:
It was not used actually. I separated them into three levels.

    ```
@Getter
    private boolean hiddenFromLAUpload;
    @Getter
    private boolean hiddenFromCTSCUpload;
    @Getter
    private boolean hiddenFromSolicitorUpload;
```